source/js/components/search-form.js

```diff
@@ -0,0 +1,108 @@
+import fetchUtils from "@degjs/fetch-utils";
+
+const searchForm = () => {
+
+
+    const formEl = document.querySelector('.js-search-form');
+    const layoutEl = document.querySelector('.js-general-content');
```

Comment:
[suggestion] Not a big deal for this project, but on larger projects, consider checking for the presence of formEl before querySelecting or delegating events to it.